app/models/loader/types.rb

```diff
@@ -96,43 +96,58 @@ def create_resource!
           resource.annotations_dataset.import(%i(resource_id name value), records)
         end
       end
-      
+
       def resource
         ::Resource[resourceid]
       end
     end
-      
+
     class Record < Types::Base
       include CreateRole
       include CreateResource
-      
+
+      def verify
+        message = "Unsupported verify for entity #{self}"
```

Comment:
I'm not sure what this means, should this be instead:
```
"#{self} doesn't implement the `#verify` method in `loader/types.rb`.
```

Reply:
I was said to wait for Shuli for proper error message - I will suggest her yours

Reply:
This seems to be okay. 
I thought of something more simple:

'Verify' method does not exist for #[self]